apiserver/pkg/http/client.go

```diff
@@ -9,8 +9,10 @@ import (
 	"strconv"
 
 	api "github.com/ray-project/kuberay/proto/go_client"
+
 	rpcStatus "google.golang.org/genproto/googleapis/rpc/status"
 	"google.golang.org/protobuf/encoding/protojson"
+	klog "k8s.io/klog/v2"
```

Comment:
A common pattern (I've seen personally) is to separate package import into three sections:
- standard packages
- repo packages
- third-party packages